.tekton/kueue-operator-pull-request.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     pipelinesascode.tekton.dev/cancel-in-progress: "true"
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"
+    pipelinesascode.tekton.dev/on-path-change-ignore: "[bundle/***, .tekton/kueue-bundle-pull-request.yaml, bundle.Dockerfile]"
+    pipelinesascode.tekton.dev/on-path-change-ignore: "[fbc/***, .tekton/kueue-fbc-418-pull-request.yaml]"
```

Comment:
You are defining the same key twice. I would be surprised that it actually appends this.

Reply:
You are probably right.